app/css/module.css

```diff
@@ -0,0 +1,23 @@
+html{
+	background: black;
+}
+
+body {
+	background: #21BFFF;
+
+}
+td {
+	color: #062ACC;
+}
+
+
+fieldset {
+	font-size: 0.75em;
+	color: #062ACC;
+}
+
+button, input{
+	border: none;
+}
+
+
```

Comment:
All these CSS files should now be Sass files, and the directory they live in should be called sass/ or styles/ or something like that, not css/